_data/publications/jimenez-rescue-hpc18.yml

```
@@ -0,0 +1,6 @@
+title: "Spotting Black Swans With Ease: The Case for a Practical Reproducibility Platform"
+link: http://doi.org/10.5281/zenodo.1488346
+date: Nov 11, 2018
+citation: Ivo Jimenez, Carlos Maltzahn, st Workshop on Reproducible, Customizable and Portable Workflows for HPC (ResCuE-HPC’18, co-located with SC’18), Dallas, TX, November 11, 2018.
```

```suggestion
citation: "Ivo Jimenez, Carlos Maltzahn, st Workshop on Reproducible, Customizable and Portable Workflows for HPC (ResCuE-HPC’18, co-located with SC’18), Dallas, TX, November 11, 2018."
```

I don't think this is required to have quotes, but nice to have.